rel/ci-config.ini

```diff
@@ -3,3 +3,8 @@
 [amqp]
 uri = "amqp://guest:guest@127.0.0.1:5672"
 pool_size = 10
+
+[log]
+syslog=debug
+console=debug
```

Comment:
i prefer `debug` for now, wouldn't caught up the order thing without it. lets keep it for now